app/database/result_store_propagate.go

```diff
@@ -359,3 +361,68 @@ func (s *ResultStore) propagate() (err error) {
 
 	return nil
 }
+
+// setResultsPropagationFromResultsPropagateItems inserts results_propagate rows from results_propagate_items.
+func (s *DataStore) setResultsPropagationFromResultsPropagateItems() {
+	const maxInserts = 5000
+
+	const fromResultsWithItemIDToPropagateQueryPart = `
+		FROM results
+		WHERE item_id IN (SELECT DISTINCT item_id FROM results_propagate_items ORDER BY item_id)
```

Comment:
why `DISTINCT` ? `item_id` is already a PK on results_propagate_items

Reply:
fixed, indeed